gobblin-core/src/main/java/gobblin/source/extractor/extract/jdbc/JdbcExtractor.java

```diff
@@ -317,11 +317,13 @@ public void extractMetadata(String schema, String entity, WorkUnit workUnit) thr
           JsonObject jsonObject = gson.fromJson(jsonStr, JsonObject.class).getAsJsonObject();
           targetSchema.add(jsonObject);
           headerColumns.add(targetColumnName);
+          sourceColumnName = "`" + sourceColumnName + "`";
```

Comment:
What about case-sensitivity? If you put double quotes around lowercase identifiers in Oracle, you might not get the desired results.